src/Content/RichContent.js

```diff
@@ -0,0 +1,21 @@
+import { createElement, Component } from 'react';
+import { string } from 'prop-types';
+
+export default class RichContent extends Component {
+    static propTypes = {
+        sanitizedRawHtml: string.isRequired
+    };
+
+    render() {
+        return (
+            this.props.children || (
+                <span
```

Comment:
This should probably be a fragment, since the contents below the `span` could be block level elements

Reply:
Good eye, I'll make that change.

Reply:
Well, turns out you can't  `dangerouslySetInnerHTML` on a Fragment, so it'll have to be a wrapper element. But I'll let 'em specify what it is, and allow props spread.